etc/docker/scripts/voila_wrapper.sh

```diff
@@ -1,4 +1,6 @@
-#!/bin/bash
+#!/bin/bash --login
+set +euo pipefail
+conda activate voilaenv
```

Comment:
There is no conda in Voila environment, it's installed to environment of jupyter kernel. So I think changes to this file should be reverted.